exporter/kafkaexporter/config.go

```diff
@@ -74,5 +79,29 @@ var _ config.Exporter = (*Config)(nil)
 
 // Validate checks if the exporter configuration is valid
 func (cfg *Config) Validate() error {
+	switch strings.ToLower(cfg.Compression.Codec) {
+	case "none":
+		if cfg.Compression.Level != defaultCompressionLevel {
```

Comment:
is there a reason to check the compression level here if compression is set to none?

Reply:
Hi @codeboten, I think this is a syntax problem. It's weird for me if the user doesn't specify the compression method but chooses a compression level. What do you think?

Reply:
I don't feel strongly either ways, it's likely not a scenario that users will run into frequently. As a user, I would find it strange to get an error message about something I configured that has no impact since i've not configured something else but that's just my preference.

Reply:
On the other side, if you forgot to add the codec, this may help find the problem faster :)